docs/TROUBLE.md

```diff
@@ -0,0 +1,175 @@
+
+Each numbered topic is structured as a test procedure with expected results followed by a list of things to check or tests to perform to help locate the problem if the expected test results are not observed.
+
+Note also that we don't mention things to check which were demonstrated to be correctly configured in previous topics.
+
+Some notation and terminology:
+
+- We use "Tx module" to refer to a mLRS radio module in the Tx or transmitter role.  In the use case assumed here, it will be installed in a OpenTX or EdgeTX radio handset.
+
+- We use "receiver" to refer to a mLRS radio module used in the receiver or Rx role.  In this use case, it will be mounted on the vehicle and connected to the flight controller.
+
+- We use lower case "rx" when refering to the receive pad or pin of a serial port connection.
+
+- We use lower case "tx" when refering to the transmit pad or pin of a serial port connection.
+
+## 1. Confirm successful firmware install
+
+In most cases, you can confirm that mLRS firmware is installed by observing the LED.  When either an receiver or Tx module is powered on by itself, a LED (usually red) should flash at 2 Hz (twice per second, on for 1/4 second then off for 1/4 second repeatedly).
+
+- If you don't see any LED activity, check power connections and voltage
+- If you don't see the expected LED pattern, follow the mLRS firmware install [documentation](https://github.com/olliw42/mLRS-docu?tab=readme-ov-file#stm32-hardware) for your hardware.
+- If you have trouble reliably flashing via ST-Link, try shorter wires, 20cm or less.
+
+## 2. Confirm operation of the mLRS LUA script
+
+Carefully install your Tx module into your OpenTX or EdgeTX radio.  Some Tx modules can be fussy on pin alignment;  Don't use excessive force or you may break something.  Follow the [instructions](LUA.md) to configure your radio and install the mLRS LUA script. Then, confirm you can talk to the Tx module via the LUA script.  You should see the TX module firmware version reported at the top of the screen.
+
+- If the LUA script does not work, ensure you have installed the correct version.  There is a separate version for radios with a lower resolution black and white screens.
+
+- Inspect the Tx module connector and ensure your Tx module is fully seated in the bay.
+
+- Check that the external CRSF baud rate is set to 400k; this is the only speed supported by mLRS.
+
+- If your Tx module is the older version of the [R9M](FRSKY_R9.md#r9m-versions), don't forget to do the Inverter Mod.
+
+## 3. Confirm radio link between Tx module and receiver is connected
+
+When pre-compiled firmware or firmware built with the same bind phrase is installed on new hardware or after erasing the flash for both the Tx module and the Rx, then they should already be bound.
```

Comment:
This reads funny to me, the heading is talking about 'connected' / 'connection' and here we say bind.  Perhaps 'After erasing the flash and flashing pre-compiled firmware the Tx module and receiver should connect automatically as they will be configured with the same bind phrase'.

IMO note about firmware built with same bind phrase could be removed, wouldn't think this to be a common thing.

Reply:
I'm was trying to be technically correct and somewhat complete, but Olli also didn't like the extra detail here so I'll try again.

How about "When custom firmware with the same bind phrase or the pre-compiled firmware is installed on new or freshly erased Receiver and Tx modules, they should already be bound."

Reply:
I suppose I still prefer something like this for the ending '...they should connect as they will already be bound with the same bind phrase.'